README.md

```diff
@@ -2,31 +2,183 @@
+System.out.println("The maximum integer is " + maximum);
+```
+
+Let's break down the new statements involving the `Math` class:
+
+`int minimum = Math.min(firstInt, secondInt);`
+
+We see that the variables `firstInt` and `secondInt` are assigned to 28 and 4
+respectively. The `min()` method can be invoked by using dot notation with the
+class name: `Math.min()`. By providing the two `int` variables as parameters,
+the `Math` class will then find the smaller of the two values and return the
+smallest `int`. In this case, it would return the value of `secondInt` or 4.
+
+`int maximum = Math.max(firstInt, secondInt);`
+
+The `max()` method can be invoked the same way with dot notation: `Math.max()`.
+By providing the two `int` variables as parameters, the Math class will then
+find the larger of the two values and return the largest `int`. In this case, it
+would return the value of `firstInt` or 28.
+
+## Absolute Value
+
+Another method of the `Math` class is the absolute value method. Absolute value
+is a non-negative value that describes how distant a number is from 0. For
+example, 0 has 0 distance from itself so its absolute value would be 0. Whereas
+-200 has a distance of 200 from 0. The same can be said for the positive
+number 200. Both have an absolute value of 200.
+
+```java
+// abs() with int data type
+int firstInt = -28;
+
+int absValue = Math.abs(firstInt);
+System.out.println("The absolute value of " + firstInt + " is " + absValue);
+```
+
+The `abs()` method, which is the absolute value method, takes one parameter.
+It can be an `int`, `double`, `float`, or `long`. In this case, it takes in an
+`int` so it will return an absolute value of the parameter as an `int`.
+Therefore, the expression `Math.abs(firstInt)` will evaluate to 28.
+
+## Ceiling and Floor
+
+Sometimes we want to round decimal values to the nearest whole number. This is
+where the `ceil()` and `floor()` methods come in from the `Math` class! The
+`ceil()` method takes a `double` as a parameter and will perform the
+ceiling on it (round the number up). The `floor` method also takes a `double`
+as a parameter and will perform the floor on it (round the number down).
+
+```java
+double x = 8.25;
+
+double ceilingValue = Math.ceil(x);
+System.out.println("The ceiling is " + x);
+
+double floorValue = Math.floor(x);
+System.out.println("The floor is " + x);
+```
+
+To invoke the `ceil()` method, we can use dot notation: `Math.ceil()` and
+provide it the variable `x` which is of type `double`. We then can store the
+return value in the `double ceilingValue`. Note that the `ceil()` method returns
+a `double`. The expression, `Math.ceil(x)` will then evaluate to 9.0. For the
+`floor()` method, we will also make use of dot notation and also provide it the
+`double` parameter. We'll store the floor value in the variable `floorValue`.
+Again, notice that the `floor()` method also returns a `double`. The floor
+expression, `Math.floor(x)` will evaluate to 8.0.
+
+## Random
```

Comment:
Would it make sense to discuss the [Random](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Random.html#nextDouble()) class and show them how to generate integer, float, and double numbers?

Reply:
I think for organization purposes, it would be best to have a separate smaller lesson on the Random class if we decide it is needed. I'd like to keep this one more strictly based on the Math class itself. I could go over how to convert this though to a random integer though if you think that would be helpful for students to know?

What do you think?

Reply:
@AlveeM - Please consider the new changes that I added here. I will add a lesson on the Random class now.